packages/playwright-test/src/common/config.ts

```diff
@@ -164,7 +164,7 @@ export class FullProjectInternal {
       grep: takeFirst(projectConfig.grep, config.grep, defaultGrep),
       grepInvert: takeFirst(projectConfig.grepInvert, config.grepInvert, null),
       outputDir: takeFirst(configCLIOverrides.outputDir, pathResolve(configDir, projectConfig.outputDir), pathResolve(configDir, config.outputDir), path.join(throwawayArtifactsPath, 'test-results')),
-      repeatEach: takeFirst(configCLIOverrides.repeatEach, projectConfig.repeatEach, config.repeatEach, 1),
+      repeatEach: takeFirst(projectConfig.repeatEach, config.repeatEach, 1),
```

Comment:
Add a comment and point to where the counterpart is? Otherwise, when I'll refactor it, I'll normalize to include cli overrides.